doc/proposals/rlp-target-gateway-resource.md

```diff
@@ -104,35 +146,76 @@ Proposed new design:
 
 ```yaml
 #  The filter’s behaviour in case the rate limiting service does not respond back. When it is set to true, Envoy will not allow traffic in case of communication failure between rate limiting service and the proxy.
-failureModeDeny: true 
-rateLimitPolicies:
-- name: some_name # policy name
-  rateLimitDomain: some_value # RLS protocol domain value
-  upstreamCluster: rate-limit-cluster # Limitador address reference
-  hostnames: # hostnames
-    - '*.toystore.com'
-  gatewayActions: # rate limit configuration list
-  - rules:
-    - paths: ["/admin/toy"]
-      hosts: ["cars.toystore.com"]
-      methods: ["POST", "DELETE"]
-    configurations:
-    - actions:
-      - generic_key:
-          descriptor_value: yes
-          descriptor_key: admin
+failureModeDeny: true
```

Comment:
nit: let's follow the same format: `failure_mode_deny`

Reply:
:+1: 

the actual implementation is already like that https://github.com/Kuadrant/wasm-shim/pull/19/files#diff-f40c086c43dffda8b3394ef8a513eb53944192a302db38fec7b224035388c668R89